app/services/request_password_reset.rb

```diff
@@ -2,20 +2,14 @@
   def perform
     # To prevent revealing account existence, we don't treat a
```

Comment:
I think that as long as there is a test for `PasswordResetEmailForm` that includes some of that context, we should be OK. 

I've seen features like this in other apps and tried to change them, but as soon as the specs failed I understood why the app was not checking for the existence of a user (if the specs were written clearly, at least...)